server/core/config/valid/global_cfg.go

```diff
@@ -129,6 +132,18 @@ var DefaultPlanStage = Stage{
 	},
 }
 
+// DefaultImportStage is the Atlantis default import stage.
+var DefaultImportStage = Stage{
+	Steps: []Step{
+		{
+			StepName: "init",
+		},
+		{
+			StepName: "import",
+		},
```

Comment:
import requires init before executing plan. So the step needs to run init first.